pom.xml

```diff
@@ -43,7 +43,7 @@ THE SOFTWARE.
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Maven+Project+Plugin</url>
 
   <properties>
-    <mavenInterceptorsVersion>1.6</mavenInterceptorsVersion>
+    <mavenInterceptorsVersion>1.7-SNAPSHOT</mavenInterceptorsVersion>
```

Comment:
Is this relying on a particular commit or PR in that project? I find it useful to note that right in the PR diff, where it is available to reviewers:

```xml
<mavenInterceptorsVersion>1.7-SNAPSHOT</mavenInterceptorsVersion> <!-- TODO …URL… -->
```